static/templates/user_profile_modal.hbs

```diff
@@ -81,6 +81,12 @@
             </div>
 
             <div class="tabcontent" id="user-profile-groups-tab">
+                <p class="manage_user_group">
+                    <a href="#organization/user-groups-admin">
+                        <i class="fa fa-cog" aria-hidden="true"></i>
+                        {{t 'Manage user groups'}}
```

Comment:
@alya do we want this label to depend on whether the current user is an organization administrator or otherwise has permissions to manage user groups?

Reply:
Ah, good idea! Perhaps "View user groups" when the user does not have management permissions?

Reply:
Yeah, that sounds right.